src/main/java/com/dev/cinema/model/Ticket.java

```diff
@@ -0,0 +1,53 @@
+package com.dev.cinema.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "tickets")
+public class Ticket {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long ticketId;
+
+    @ManyToOne
+    private User user;
+
+    @OneToOne
```

Comment:
```suggestion
    @ManyToOne
```
we can have a lot of tickets for one session otherwise you would be sitting all alone )

Reply:
It would be great for me)

Reply:
Done